GW2EIEvtcParser/EncounterLogic/Fractals/FractalLogic.cs

```diff
@@ -34,235 +34,231 @@ protected FractalLogic(int triggerID) : base(triggerID)
+                {
+                    EnvironmentDecorations.Add(new CircleDecoration(120, (previousPulse, pulse), Colors.Blue, 0.1, new PositionConnector(effect.Position)).UsingGrowingEnd(pulse));
+                    previousPulse = pulse;
+                    pulse += pulseDuration;
                 }
             }
         }
+    }
 
-        /// <summary>
-        /// Add AoE decorations based on the distance to the caster. <br></br>
-        /// Used for Caustic Barrage (Siax, Ensolyss) and Solar Bolt (Skorvald).
-        /// </summary>
-        /// <param name="log">The log.</param>
-        /// <param name="environmentDecorations">The decorations list.</param>
-        /// <param name="effectGUID">The <see cref="EffectGUIDs"/>.</param>
-        /// <param name="target">The target casting.</param>
-        /// <param name="distanceThreshold">Threshold distance of the effect from the caster.</param>
-        /// <param name="onDistanceSuccessDuration">Duration of the AoE effects closer to the caster.</param>
-        /// <param name="onDistanceFailDuration">Duration of the AoE effects farther away from the caster.</param>
-        protected static void AddDistanceCorrectedOrbDecorations(ParsedEvtcLog log, CombatReplayDecorationContainer environmentDecorations, string effectGUID, TargetID target, double distanceThreshold, long onDistanceSuccessDuration, long onDistanceFailDuration)
+    /// <summary>
+    /// Add AoE decorations based on the distance to the caster. <br></br>
+    /// Used for Caustic Barrage (Siax, Ensolyss) and Solar Bolt (Skorvald).
+    /// </summary>
+    /// <param name="log">The log.</param>
+    /// <param name="environmentDecorations">The decorations list.</param>
+    /// <param name="effect">The <see cref="EffectGUIDs"/>.</param>
+    /// <param name="target">The target casting.</param>
+    /// <param name="distanceThreshold">Threshold distance of the effect from the caster.</param>
+    /// <param name="onDistanceSuccessDuration">Duration of the AoE effects closer to the caster.</param>
+    /// <param name="onDistanceFailDuration">Duration of the AoE effects farther away from the caster.</param>
+    protected static void AddDistanceCorrectedOrbDecorations(ParsedEvtcLog log, CombatReplayDecorationContainer environmentDecorations, GUID effect, TargetID target, double distanceThreshold, long onDistanceSuccessDuration, long onDistanceFailDuration)
+    {
+        if (!log.AgentData.TryGetFirstAgentItem(target, out AgentItem agent))
         {
-            if (!log.AgentData.TryGetFirstAgentItem(target, out AgentItem agent))
-            {
-                return;
-            }
-            if (log.CombatData.TryGetEffectEventsByGUID(effectGUID, out IReadOnlyList<EffectEvent> effects))
+            return;
+        }
+        if (log.CombatData.TryGetEffectEventsByGUID(effect, out var events))
+        {
+            foreach (EffectEvent @event in events)
```

Comment:
I'm seeing such renaming patterns here and there, what is the reasoning?

Reply:
`effectGUID` to `effect` ? The information about it being a GUID is now encoded in the type, so no need to have it in the variable name.

Reply:
I was rather curious about the "events" and "@event" renaming ^^ Sounded a little bit generic, I would have gone for "effectEvents" and "effectEvent" for better clarity

Reply:
Ah - well, in some cases there where name collisions form the effectGUID(s) -> effects rename (since its now a distinct type) 
Then i realized that the old out param `effects` used in a bunch of places is straight up named incorrectly. It doesn't give you multiple effects, it gives you the events of that one specific effect.
`effectEvents` is certainly even more verbose, if you want me to change it to that we can do so np.

Reply:
@event makes it easy to track it down, it can be done later if you don't want to go through some files again.

Reply:
to be fully honest im not sure i even changed all of them. I only changed the ones that were causing name collisions or where right under my nose. So if you wanted to clean that up it would need to be more methodical than just searching  the name.